pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -56,13 +61,14 @@ type cniNetworkPlugin struct {
 	sync.RWMutex
 	defaultNetwork *cniNetwork
 
-	host        network.Host
-	execer      utilexec.Interface
-	nsenterPath string
-	confDir     string
-	binDirs     []string
-	cacheDir    string
-	podCidr     string
+	host         network.Host
+	execer       utilexec.Interface
+	nsenterPath  string
```

Comment:
@AlexeyPerevalov nsenterPath is now unused.

Reply:
yes, done